### PR TITLE
AAE-12770 add markdown formatting

### PR DESCRIPTION
**Pull request by atchertchian**

### CODE REVIEW

**Review by gicappa — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -12,96 +11,103 @@ Activiti
...
+  - Click tree item _Compiler_, textfield _Resource patterns_: change to `!?*.java` (remove other content)
+
+- Avoid a `StackOverflowError` when building
+
+  - Open menu _File_, menu item _Settings_ or menu _IntelliJ IDEA_, menu item _Preferences..._ if on a Mac
+  - Click tree item _Compiler_, tree item _Java Compiler_, textfield _Additional command line parameters_
+  - Add `-J-Xss1024k`
+
+- Recommended code style: use the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html) with editorconfig
+
```
Comment (erdemedeiros) [inline README.md:39]: It's out of the scope of this PR, but is this still true?

**Review by erdemedeiros — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (atchertchian) [reply README.md:39]: No indeed, and the whole folder at https://github.com/Activiti/Activiti/tree/develop/ide-configuration should be reviewed (with AAE-13036)

**Pull request merged**